lib/actions/apiV2.js

```diff
@@ -19,12 +34,46 @@ import {
   receivedNearbyStopsError,
   receivedNearbyStopsResponse,
   receivedVehiclePositions,
-  receivedVehiclePositionsError
+  receivedVehiclePositionsError,
+  rememberSearch,
+  routingError,
+  routingRequest,
+  routingResponse,
+  updateOtpUrlParams
 } from './api'
+import { ItineraryView, setItineraryView } from './ui'
+import { rememberPlace } from './user'
 import { zoomToPlace } from './map'
 
+const { generateCombinations, generateOtp2Query } = coreUtils.queryGen
+const { getTripOptionsFromQuery, getUrlParams } = coreUtils.query
 const LIGHT_GRAY = '666666'
 
+function formatRecentPlace(place) {
+  return convertToPlace({
+    ...place,
+    icon: 'clock-o',
+    id: `recent-${randId()}`,
+    timestamp: new Date().getTime(),
+    type: 'recent'
+  })
+}
+
+function formatRecentSearch(state, queryParamData) {
+  return {
+    id: randId(),
+    query: getTripOptionsFromQuery(
+      { ...state.otp.currentQuery, queryParamData },
+      true
+    ),
+    timestamp: new Date().getTime()
+  }
+}
+
+function isStoredPlace(place) {
+  return ['home', 'work', 'suggested', 'stop'].indexOf(place.type) !== -1
```

Comment:
This might need some work...

Reply:
This code doesn't look familiar to me at all

Reply:
What does this have to do with the mode selector?

Reply:
Actually, since the functions are no longer used anywhere else, I just deleted them from the api.js file. I think at this point any further refactoring can be left to the future OTPv1 removal PR, which will change all of this.